src/components/MenuBar.vue

```diff
@@ -77,8 +82,21 @@ export default {
 	data() {
 		return {
 			searchQuery: "",
+			avatarUrl: "", // Inicializa o avatarUrl
 		};
 	},
+	async created() {
+		const { data: user } = await supabase.auth.getUser();
+
+		// Tenta carregar a URL do avatar do localStorage
+		const savedUser = JSON.parse(localStorage.getItem("user"));
+		if (savedUser && savedUser.avatarUrl) {
+			this.avatarUrl = savedUser.avatarUrl; // Carrega do localStorage
+		} else if (user && user.user_metadata) {
```

Comment:
https://github.com/PatrickSant0s/Link-Buy/pull/12/files#r1801014408
Mesma coisa do comentário acima:

```suggestion
		} else if (user?.user_metadata) {
```

Reply:
![image](https://github.com/user-attachments/assets/40a2d584-f80d-4381-b945-0c4d5018c0cc)
ajustado